### PR TITLE
modify mish op and add mish api

### PR DESCRIPTION
**Pull request by wangxinxin08**
### PR types
New features

### PR changes
APIs

### Describe
add mish op and api in dygraph and remove original mish op, the inference speed of ppyolov2 r50vd dcn before and after this PR can be seen as follows:
56a68124072fdd21ad4212e2ac4c82e4

### CODE REVIEW

**Review by XieYunshen — APPROVED** [review]
LGTM

**Review by heavengate — APPROVED** [review]

**Review by XiaoguangHu01 — APPROVED** [review]
LGTM

File: python/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -1174,6 +1174,47 @@ def swish(x, name=None):
...
+
+    Returns:
+        A Tensor with the same data type and shape as ``x`` .
+
+    Examples:
+        .. code-block:: python
+
+            import paddle
+            import paddle.nn.functional as F
+            import numpy as np
```
Comment (TCChenlong) [inline python/paddle/nn/functional/activation.py:1203]: 不用np 直接 paddle.to_tensor 即可
下同

**Review by TCChenlong — APPROVED** [review]
LGTM

File: python/paddle/nn/functional/activation.py (hunk 1 above)
Comment (wangxinxin08) [reply python/paddle/nn/functional/activation.py:1203]: ok

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**